cfme/tests/intelligence/reports/test_menus.py

```diff
@@ -171,3 +169,58 @@ def test_rbac_move_custom_report(
             type=folder, subtype=subfolder, menu_name=get_custom_report.menu_name
         )
         assert rbac_report.exists
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1676638, 1731017])
+@pytest.mark.parametrize("group", GROUPS)
+def test_reports_menu_with_duplicate_reports(appliance, request, group, report_menus):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        initialEstimate: 1/10h
+        startsin: 5.9
+        setup:
+            1.Create a custom report or copy an existing report.
+            2. Move the custom report to 'Tenants > Tenant Quotas' menu in user's current group
```

Comment:
- Need space here:  ```1. Create a custom .....```
- Also I can see this test case ignored for ```5.10``` but it's ```startsin``` version is ```5.9```. I think it should be ```5.11```